test/hotspot/jtreg/serviceability/EarlyDynamicLoad/libEarlyDynamicLoad.c

```diff
@@ -34,18 +36,38 @@
 #endif // WINDOWS
 
 static void JNICALL VMStartJcmd(jvmtiEnv* jvmti, JNIEnv* env) {
+    const char* jcmd_path = getenv("JCMD_PATH");
+    FILE* file = fopen(jcmd_path, "r");
+    if (file == NULL) {
+        printf("%s does not exist", jcmd_path);
+        return;
+    }
+    fclose(file);
+
     char cmd[256];
-    snprintf(cmd, sizeof(cmd), "%s %d JVMTI.agent_load some.jar", getenv("JCMD_PATH"), PID());
+    snprintf(cmd, sizeof(cmd), "%s %d help", jcmd_path, PID());
```

Comment:
Why are you running the help command instead of `agent_load` ???

Reply:
See my [earlier comment](https://github.com/openjdk/jdk/pull/27766/#issuecomment-3413186002), I'm debugging failures on Windows so my code isn't in a reviewable state. Thus why I marked the PR as draft.

I'll make it ready again as soon as I fix the tests on Windows.